docs/dev/libvirt/README.md

```diff
@@ -286,6 +286,38 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### Console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console depends on the route which is created by auth operator.
+To make it works we need to first create the manifests and edit the `domain` for ingress config, before directly creating the cluster.
+
+Following steps are required to have console operator up and working.
```

Comment:
I think the previous two sentences cover this.  If you want to make it easier to blindly find this documentation, maybe include an example of the error message that eventually bubbles out if you don't take these steps?